docs/server/kubernetes-operator/getting-started/installation.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Installation
+order: 2
+---
+
+This section covers the various aspects of installing the Operator.
+
+::: important
+For production deployments, it is _highly_ recommended to follow the best practices described in this guide.
+:::
+
+## Prerequisites
+
+::: tip
+To get the best out of this guide, a basic understanding of [Kubernetes concepts](https://kubernetes.io/docs/concepts/) is essential.
+:::
+
+Before installing and executing the Operator, the following requirements should be met:
+
+* Access to a Kubernetes cluster with a minimum version of `v1.23.1+`.
+* Sufficient permissions to deploy the Operator and Custom Resource Definitions (CRDs).
+* The following CLI tools are installed and configured to interact with your Kubernetes cluster. This means the tool must be accessible from your shell's `$PATH`, and your `$KUBECONFIG` environment variable must point to the correct Kubernetes configuration file:
+  * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+  * [k9s](https://k9scli.io/topics/install/)
+* The [Helm 3 CLI](https://helm.sh/docs/intro/install/) tool is installed and configured to interact with your Kubernetes cluster.
+* Credentials for the Operator artifact repository (please [contact us](https://www.kurrent.io/contact) for more information). References to the credentials appear as `$username` and `$password` in the guide.
+
+## Helm Repository
+
+The Operator deployment process is managed via Helm. The following Kurrent repository must be configured using the command:
+
+```bash
+helm repo add kurrentdb-operator-repo \
+  'https://packages.eventstore.com/basic/kurrentdb-operator/helm/charts/' \
+  --username $username \
+  --password $password
+```
+
+## Custom Resource Definitions (CRDs)
+
+The following resource types are supported by the Operator:
+- [KurrentDB](resource-types.md#kurrentdb)
+- [KurrentDBBackup](resource-types.md#kurrentdbbackup)
+
+Since CRDs are managed globally by Kubernetes, special care must be taken to install them.
```

Comment:
Exactly what type of special care should the user take? Suggest avoid being giving vague directions..